frontend/demo/component/button/button-labels.ts

```diff
@@ -0,0 +1,93 @@
+import 'Frontend/demo/init'; // hidden-full-source-line
+
+import { css, html, LitElement, customElement } from 'lit-element';
+import '@vaadin/vaadin-text-field/vaadin-text-field';
+import '@vaadin/vaadin-button/vaadin-button';
+import '@vaadin/vaadin-ordered-layout/vaadin-vertical-layout';
+import '@vaadin/vaadin-ordered-layout/vaadin-horizontal-layout';
+import { TextFieldElement } from '@vaadin/vaadin-text-field/vaadin-text-field';
+import { applyTheme } from 'Frontend/generated/theme';
+
+@customElement('button-labels')
+export class Example extends LitElement {
+  constructor() {
+    super();
+    // Apply custom theme (only supported if your app uses one)
+    applyTheme(this.shadowRoot);
+  }
+
+  static get styles() {
+    return css`
+      vaadin-horizontal-layout {
+        align-items: baseline;
+      }
+      vaadin-button {
+        margin-left: 0.5rem;
+      }
+    `;
+  }
+
+  render() {
+    return html`
+      <!-- tag::snippet[] -->
+      <vaadin-vertical-layout>
+        <vaadin-horizontal-layout>
+          <vaadin-text-field
+            id="primary-email"
+            label="Primary email address"
+            value="foo@example.com"
+          ></vaadin-text-field>
+          <vaadin-button arial-label="Remove primary email address" @click=${this.clearPrimaryEmail}
```

Comment:
Check that quotes are added around all attribute values.

```suggestion
          <vaadin-button arial-label="Remove primary email address" @click="${this.clearPrimaryEmail}"
```

Reply:
Done.